java/src/jmri/jmrit/consisttool/ConsistComboBox.java

```diff
@@ -0,0 +1,95 @@
+package jmri.jmrit.consisttool;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import javax.swing.JComboBox;
+
+import jmri.*;
+
+import org.jdom2.JDOMException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ *
+ * @author Lionel Jeanson Copyright (c) 2023
```

Comment:
The copyright notice is usually below the description of the class.

```
/**
 * A JComboBox with JMRI consists
 * Entries can be a String or a DccLocoAddress 
 * 
 * @author Lionel Jeanson Copyright (c) 2023
 */
```

Reply:
ok, done, will be in next push.